app/Service/Kata7/StringEndService.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace App\Service\Kata7;
+
+class StringEndService
+{
+    /**
+     * @param string $stringValue
+     * @param string $stringEnding
+     * @return string
+     */
+    public function checkIfStringMatchesTheGivenEnding(string $stringValue, string $stringEnding): string
+    {
+        $isStringMatchingTheGivenEnding = str_ends_with($stringValue, $stringEnding);
```

Comment:
I would place `str_ends_with($stringValue, $stringEnding);` in the if condition immediately instead of putting it in a variable